_opensearch/install/plugins.md

```diff
@@ -64,6 +64,27 @@ bin/opensearch-plugin list
     </tr>
   </thead>
   <tbody>
+  <tr>
+  <tr>
+    <td>2.0.0.0</td>
+    <td>
+      <pre>opensearch-alerting                  2.0.0.0
+opensearch-anomaly-detection         2.0.0.0
+opensearch-asynchronous-search       2.0.0.0
+opensearch-cross-cluster-replication 2.0.0.0
+opensearch-index-management          2.0.0.0
+opensearch-job-scheduler             2.0.0.0
+opensearch-knn                       2.0.0.0
+opensearch-ml                        2.0.0.0
+opensearch-notifications             2.0.0.0
+opensearch-observability             2.0.0.0
+opensearch-performance-analyzer      2.0.0.0
+opensearch-reports-scheduler         2.0.0.0
+opensearch-security                  2.0.0.0
+opensearch-sql                       2.0.0.0
+</pre>
+</td>
+</tr>
```

Comment:
Hi @Naarcha-AWS we are missing 1.3.2.0 plugin info for opensearch as well.
Since you add them for dashboards.
Can you add here for opensearch as well?
Thanks.

Reply:
1.3.2 does not have notifications be aware.

Reply:
Done